tests/system/providers/amazon/aws/example_bedrock.py

```diff
@@ -16,17 +16,61 @@
 # under the License.
 from __future__ import annotations
 
+import json
 from datetime import datetime
 
+from botocore.exceptions import ClientError
+
+from airflow.decorators import task
 from airflow.models.baseoperator import chain
 from airflow.models.dag import DAG
-from airflow.providers.amazon.aws.operators.bedrock import BedrockInvokeModelOperator
+from airflow.providers.amazon.aws.hooks.bedrock import BedrockHook
+from airflow.providers.amazon.aws.operators.bedrock import (
+    BedrockCustomizeModelOperator,
+    BedrockInvokeModelOperator,
+)
+from airflow.providers.amazon.aws.operators.s3 import (
+    S3CreateBucketOperator,
+    S3CreateObjectOperator,
+    S3DeleteBucketOperator,
+)
+from airflow.providers.amazon.aws.sensors.bedrock import BedrockCustomizeModelCompletedSensor
+from airflow.utils.trigger_rule import TriggerRule
 from tests.system.providers.amazon.aws.utils import SystemTestContextBuilder
 
-sys_test_context_task = SystemTestContextBuilder().build()
+# Externally fetched variables:
+ROLE_ARN_KEY = "ROLE_ARN"
+sys_test_context_task = SystemTestContextBuilder().add_variable(ROLE_ARN_KEY).build()
 
 DAG_ID = "example_bedrock"
+
+# Creating a custom model takes nearly two hours. If SKIP_LONG_TASKS is True then set
```

Comment:
This is a shame, is there no way to speed this up, or even cancel it once we ensure it has started correctly?

Reply:
Negative.   That is the smallest possible job I can come up with (feeding the smallest model one datapoint to learn), there is no cancel command, and it can not be deleted until it is in a terminal state, whether that is a successful build or a failed job.

Reply:
That's very sad. Okay, this is a nice workaround if that's the case :+1:

Reply:
I've been putting thought into using a BranchingOperator to just skip them entirely if the SKIP_LONG flag is true, rather than assign them an improbable trigger_rule.  Then maybe we can figure out some trick to get our dashboard CI to run the long version from time to time by setting an envvar or something.... I don't know, needs some thinking, but I think this is a decent compromise for the time being.